src/common/util.cpp

```diff
@@ -727,6 +750,32 @@ std::string get_nix_version_display_string()
     if (!unbound_built_with_threads())
       MCLOG_RED(el::Level::Warning, "global", "libunbound was not built with threads enabled - crashes may occur");
 
+#ifdef WIN32
+    OSVERSIONINFOEX osvi;
+    BOOL bOsVersionInfoEx;
+    DWORD dwType;
+
+    ZeroMemory(&osvi, sizeof(OSVERSIONINFOEX));
+
+    osvi.dwOSVersionInfoSize = sizeof(OSVERSIONINFOEX);
+    bOsVersionInfoEx = GetVersionEx((OSVERSIONINFO*) &osvi);
+
+    // On Windows 10, ANSI colors were enabled by default between
+    // builds 10586 and 14393 (exclusive). If we are running on a
+    // build greater or equal than 14393, we need to explictely
+    // enable ANSI colors in the terminal.
+    if( bOsVersionInfoEx && osvi.dwMajorVersion >= 10 )
+    {
+      if ( osvi.dwMajorVersion > 10 || osvi.dwBuildNumber >= 14393 )
```

Comment:
Especially as this doesn't seem to match 0..10585.

Reply:
I inspired from the way it was handled here: https://github.com/bin-build/bin/commit/e4a038bb7f89b51f3eb794b83da6fa054d48900b
You don't need to enable it between versions 0 and 10585, as it is enabled by default. I'm ok with enabling it all the time, I don't think there would be any side effects in doing so.

Reply:
The comment imply otherwise (though doesn't say it explicitely). Moot anyway.